backend/src/routes/v1/webAde.js

```diff
@@ -12,23 +12,13 @@ const {
 // fetches the app config json for a acronym in a specified env
 webAde.get('/:webAdeEnv/:appAcronym/appConfig', [
 ], async (req, res) => {
-  // TODO: a lot of this role checking is duplicate, but as we will be moving acronym management to the DB soon all this will need to be refactored anyways
-
   // Check for required permissions. Can only fetch cfgs for the acronyms you are associated with
   // If the user has "WEBADE_CFG_READ_ALL" then they can get all
-  let acronyms = [];
-  const roles = req.user.jwt.realm_access.roles;
-  let hasReadAllRole = false;
-  if (typeof roles === 'object' && roles instanceof Array) {
-    acronyms = utils.filterAppAcronymRoles(roles);
-    hasReadAllRole = roles.includes('WEBADE_CFG_READ_ALL');
-  }
-
-  if (!hasReadAllRole) {
-    const appAcronym = req.params.appAcronym;
-    if (!acronyms.includes(appAcronym)) {
+  if (!req.user.jwt.realm_access.roles.includes('WEBADE_CFG_READ_ALL')) {
```

Comment:
Could you put that check in checkAcronymPermission?  or is that method used for checking for writing too, and this would only apply to read?

Reply:
Good point. Existing method is for write permission checking, but this logic can move over there too.
Will refactor in next pr I'm working on.